File: Redes/AtvCRuby/cliente.rb
require 'socket'

def start_server(host, port)
  client_socket = TCPSocket.new(host, port)
  
  puts 'Type your message: '
  message = gets.chomp
  client_socket.send(message.encode('utf-8'), 0)
  
  client_socket.close
end

HOST = 'localhost'
PORT = 9000       

start_server(HOST, PORT)

File: Redes/CTransporte/tcp/cliente.py
import socket

def start_server(host: str, port: int):
        client_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)

        message = input('Type your message: ')
        client_socket.connect((host, port))
        client_socket.sendall(message.encode('utf-8'))

        client_socket.close()
       
if __name__=='__main__':
    HOST = 'localhost' #ENDEREÇO IP SO SERVIDOR
    PORT = 8000 #PORTA --

    start_server(HOST,PORT)

